### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by ZackGoing** [duanzhiihao/RAPiD#6]
Total time: 18:52:17.857143, iter: 0:00:07.628324, epoch: 1:59:21.331554
[Iteration 8904] [learning rate 0.001] [Total loss 150.54] [img size 672]
level_21 total 6 objects: xy/gt 1.189, wh/gt 0.078, angle/gt 0.475, conf 28.056
level_42 total 14 objects: xy/gt 1.211, wh/gt 0.041, angle/gt 0.049, conf 42.867
level_84 total 7 objects: xy/gt 1.332, wh/gt 0.062, angle/gt 0.135, conf 40.987
Max GPU memory usage: 3.0111474990844727 GigaBytes
------------------------------------------------------------------------------------------------------
the conf is still high at iteration 8904,is it normal?

**Comment by duanzhiihao** [comment]
Yes it's normal since the confidence loss (i.e., `self.loss4obj` in the code) is the summation of conf. losses of all anchor boxes. There are thousands of anchor boxes in each image so the confidence loss is usually very large.

https://github.com/duanzhiihao/RAPiD/blob/df9a52948c60b1a8cabb5bdae68b70dab11968e4/models/rapid.py#L114

**Comment by ZackGoing** [comment]
would you like to share your train.log?Thx

**Comment by duanzhiihao** [comment]
Sorry, I don't have access to the training logs.

Did you try to visualize the predicted bounding boxes using the trained weights?

**Comment by ZackGoing** [comment]
not yet
